api_app/services/aad_authentication.py

```diff
@@ -193,7 +193,7 @@ def _get_app_auth_info(self, client_id: str) -> dict:
             raise AuthConfigValidationError(f"{strings.ACCESS_UNABLE_TO_GET_INFO_FOR_APP} {client_id}")
 
         app_info = graph_data['value'][0]
-        authInfo = {'sp_id': app_info['id']}
+        authInfo = {'sp_id': app_info['id'], 'scope_id': app_info['servicePrincipalNames'][0]}
```

Comment:
Is this the branch when we don't get scope_id from the bundle?

Reply:
Yes this flow is when we are retrieving the aad info rather than creating it

Reply:
Leave a comment clarifying this?